ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/QueryStreamWriter.java

```diff
@@ -123,11 +132,47 @@ public void write(OutputStream out) {
   }
 
   private void write(OutputStream output, GenericRow row) throws IOException {
-    objectMapper.writeValue(output, StreamedRow.row(row));
+    objectMapper.writeValue(output, StreamedRow.row(updateRow(row)));
     output.write("\n".getBytes(StandardCharsets.UTF_8));
     output.flush();
   }
 
+  private GenericRow updateRow(GenericRow genericRow) {
```

Comment:
Badly named record. It doesn't update the row passed in.

Reply:
Also, can it be static?

Reply:
changed the name to `getNeUpdatedRow`. 
Cannot be static since it uses jsonConverter indirectly.